spec/Conversions.tex

```diff
@@ -248,14 +278,29 @@ \subsection{Implicit Statement Bool Conversions}
 \item An expression of a class type is taken to be false if it is nil and is true otherwise.
 \end{itemize}
 
+\subsection{Implicit Conversions To Generic Types}
+\label{Implicit_Generic_Type_Conversions}
+\index{conversions!implicit!generic types}
+
+When the target type \chpl{T} is generic (\rsec{Generic_Types}),
+an implicit conversion is allowed when there is an instantiation
+of this type such that an implicit conversion is allowed between
+the source type and that instantiation by another rule in this section.
```

Comment:
Do you think a reader would think that this means that for a record like `record R { var x; }` that `R(int(8))` could coerce into `R(int)`? (Because it does not, today).

Reply:
Well `R(int)` is not generic, so this paragraph does not apply.

Cf. `R(int(8))` to `R(int(?))` should coerce by this rule.

Is this OK?

Reply:
I thought about `record R { var x; var y; }` in this way, but I think it is OK, because we won't have `R(int(8))` coercing to `R(int)` by the other part of the section.